tests/test_models.py

```diff
@@ -201,18 +206,30 @@ def test_opening_mask_ref(tmp_path):
 
 # Read Noise tests
 def test_make_readnoise():
-    readnoise = utils.mk_readnoise(shape=(20,20))
+    readnoise = utils.mk_readnoise(shape=(20, 20))
     assert readnoise.meta.reftype == 'READNOISE'
     assert readnoise.data.dtype == np.float32
 
     # Test validation
     readnoise_model = datamodels.RampModel(readnoise)
     assert readnoise_model.validate() is None
 
+
 def test_opening_readnoise_ref(tmp_path):
     # First make test reference file
     file_path = tmp_path / 'testreadnoise.asdf'
     utils.mk_readnoise(filepath=file_path)
     readnoise = datamodels.open(file_path)
     assert readnoise.meta.instrument.optical_element == 'F158'
     assert isinstance(readnoise, datamodels.ReadnoiseRefModel)
+
+
+def test_add_model_attribute(tmp_path):
```

Comment:
Do we need to be able to save these new attributes to the .asdf file, and read them back again?  That's worth a test, if so.

Reply:
Sure, though I don't see why that won't work. But I'll add a test.